web-ui/src/components/reviews/periods/DatePickerField.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { useRef, useEffect, useState } from 'react';
+import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
+import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
+import { DatePicker } from '@mui/x-date-pickers/DatePicker';
+import './DatePickerField.css';
+
+export default function DatePickerField({ date, setDate, label, disabled, open}) {
+
+  const [cleared, setCleared] = useState(false);
+  const launchDatePickerRef = useRef(null);
+
+  useEffect(() => {
+    if (cleared) {
+      const timeout = setTimeout(() => {
+        setCleared(false);
+      }, 1500);
+
+      return () => clearTimeout(timeout);
+    }
+    return () => {};
```

Comment:
Is line 20 really needed?

Reply:
I think this is not needed, this was from the MUI example. Deleted.